browser/brave_wallet/brave_wallet_importer_delegate_impl.cc

```diff
@@ -69,16 +69,25 @@ void BraveWalletImporterDelegateImpl::ImportFromBraveCryptoWallet(
     return;
   }
 
-  extension_registry_observer_.Observe(ExtensionRegistry::Get(context_));
+  ExtensionRegistry* registry = ExtensionRegistry::Get(context_);
+  const Extension* extension =
+      registry->GetInstalledExtension(ethereum_remote_client_extension_id);
+  // crypto wallet is not loaded
```

Comment:
nit: Crypto Wallets

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558